### DISCUSSION TITLE
How to debug in vscode?

### DISCUSSION THREAD

**Discussion by shlroland** [samchon/typia#1329] [Q&A]
related #1305 

I’m quite confused about how to debug my Node.js or Bun application in VSCode after using Typia.

**Comment by samchon** [comment]
This has been addressed by documentation PR #2000:

https://github.com/samchon/typia/pull/2000

The new guide documents the debugger paths for TypeScript-Go (`ttsc` / `ttsx`), legacy `ts-node`, `typia generate`, and Bun. The PR has passed CI and full self-review; the docs will be available after it is merged/published.

For TypeScript-Go projects, please wait for the official TS-Go-based typia release before treating the TypeScript-Go setup as the stable path.
